src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetTimelineTest.kt

```diff
@@ -155,7 +155,7 @@ class GetTimelineTest : IntegrationTestBase() {
             .hasPrisonId("BXI")
             .wasActionedBy("system")
             .hasNoActionedByDisplayName()
-            .hasContextualInfo("MDI")
+            .hasContextualInfo(mapOf("PRISON_TRANSFERRED_FROM" to "MDI"))
```

Comment:
Here the contextual info is a `Map<String, String>` - you might wonder why its not keyed on `TimelineEventContext` 🤔 

At the API its a `Map<String, String>` because we define the APIs in the swagger spec first and it generates our model classes. See the changes below to the swagger spec, but I couldn't work out how to define a map keyed by a type (rather than keyed by string)